webapp/src/selectors.ts

```diff
@@ -240,6 +240,9 @@ export const callThreadIDForCallInChannel = (state: GlobalState, channelID: stri
     return pluginState(state).calls[channelID]?.threadID || '';
 };
 
+export const callThreadIDForCurrentCall = (state: GlobalState) =>
```

Comment:
If we follow the rules of consistency I totally made up (:p) this would probably become `threadIDForCurrentCall`.
But I also realize I forgot to update `callThreadIDForCallInChannel` which probably caused the confusion.